.gitignore

```diff
@@ -1,3 +1,5 @@
 /taskcluster-base-test.conf.json
 node_modules/
 .vagrant/
+.test/
+lib/
```

Comment:
I think having only .gitignore is the safest and easiest for now.  it does mean that we ship two copies of our libraries, but that's nbd and worth the overhead to avoid the risk imo.